blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,238 @@
+def faq_fraud_charge_handler(request, responder):
+    replies = ['If you think someone used your account without your permission or notice any charge errors, tell us immediately by calling 1-800-MindMeld.']
+    responder.reply(replies)
+
+@app.handle(intent='forgot_pin')
+def faq_forgot_pin_handler(request, responder):
+    replies = ['If you\'ve forgotten your PIN or don\'t have one, please visit your local branch or contact customer service at 1-800-MindMeld to have a new PIN mailed to you.']
+    responder.reply(replies)
+
+@app.handle(intent='apply_loan')
+def faq_apply_loan_handler(request, responder):
+    replies = ['To apply for a loan or any further questions you will need to visit our website MindMeld.com/loans']
+    responder.reply(replies)  
+
+@app.handle(intent='activate_creditcard')
+def faq_activate_creditcard_handler(request, responder):
+    replies = ['You can activate your card by logging into our website and also over the phone. Just call the MindMeld card activation number 1-800-MindMeld. Follow the prompts that will trigger the activation of your card.']
+    responder.reply(replies)
+
+@app.handle(default=True)
+def default(request, responder):
+    replies = ['Sorry, I didn’t get that. Try asking about account balances or ordering checks', 
+            "I'm afraid I don't understand. You can try to ask to check your balance or pay a bill",
+            'Sorry, I do not know what that is. Try asking about card activation or reporting a stolen card', 
+            "I'm afraid I dont't understand. A few banking tasks you can try are, transferring balances and paying bills"]
+    responder.reply(replies)
+
+@app.handle(intent='credit_due_date')
+def check_due_date_handler(request, responder):
+    replies = ['Your credit card bill is due on the 15th of every month, late payments will result in a $25 fee']
+    responder.reply(replies)
+
+
+def check_amount(amount):
+    pass
+    #logic for checking if amount is available to be transferred 
+
+
+entity_form = {
+    'entities':[
+        FormEntity(
+            entity='account_type',
+            role='origin',
+            responses=['Sure. Transfer from which account?']),
+        FormEntity(
+            entity='account_type',
+            role='dest',
+            responses=['To which account?']),
+        FormEntity(
+            entity='sys_amount-of-money',
+            responses=['And, how much do you want to transfer?'])
+            #custom_eval=check_amount
+        ],
+}
+
+def check_value(responder):
+    return get(responder.slots['origin']) >= responder.slots['amount']
+
+@app.auto_fill(intent='transfer_balances', form=entity_form)
+def transfer_balances_handler(request, responder):
+	
+    for entity in request.entities:
+        if entity['type'] == 'account_type':
+            if entity['role'] == 'origin':
+                print(entity)
+                responder.slots['origin'] = entity['value'][0]['cname'] or entity['text']
```

Comment:
This will throw an error if the value isn't resolved, i.e. ``value = {}``. Instead you can try the following:

```
try:
    responder.slots['origin'] = entity['value'][0]['cname']
except KeyError as e:
    responder.slots['origin'] = entity['text']
```
or
``responder.slots['origin'] = entity['value'][0]['cname'] if len(entity['value']) > 0 else entity['text']``